runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesPlugin.java

```diff
@@ -291,24 +344,26 @@ private void updateWarning(boolean weaponSwitch)
 				{
 					if (weaponSwitch)
 					{
-						// TODO : chat message to warn players that their weapon switch also caused an unwanted attack style change
+						sendChatMessage(WEAPON_SWITCH_MESSAGE);
```

Comment:
Sending a notification should be a separate commit because it's solving a different issue.

Reply:
You mean a new pull request?

Reply:
no, commit.